src/ac_training_lab/pioreactor/on_reactor.py

```diff
@@ -0,0 +1,321 @@
+import requests
+import json
+from datetime import datetime
+import paho.mqtt.client as mqtt
+import json
+import lookhere
+import time
+
+# Author: Enrui (Edison) Lin 
+
+
+# The lookhere.py file contains the following variables:
+# broker = "your_broker_address"
```

Comment:
Like we discussed on gather town, perhaps have a checkbox labeled "Use custom credentials" that opens up a "form" like the following: https://huggingface.co/spaces/AccelerationConsortium/light-mixing/blob/main/app.py#L38. (of course making sure not to expose secrets)

Reply:
I think I am missing something, I don't see the checkbox, could you please elaborate